src/pages/Home/Home.tsx

```diff
@@ -3,6 +3,7 @@ import { quotes } from '@/components/CarouselQuote/data';
 import { CarouselTestimonial } from '@/components/CarouselTestimonial/CarouselTestimonial';
 import { testimonials } from '@/components/CarouselTestimonial/Testimonials';
 import { ProfileCard } from '@/components/ProfileCard/ProfileCard';
+import image from '/fahan.png'
```

Comment:
Ideally this should have a more descriptive name, e.g. fahanImage

Reply:
done